cumulusci/tasks/release_notes/generator.py

```diff
@@ -95,6 +103,83 @@ def _init_change_notes(self):
         return DirectoryChangeNotesProvider(self, self.directory)
 
 
+class ParentPullRequestNotesGenerator(BaseReleaseNotesGenerator):
+    """Aggregates notes from child pull requests in to a parent pull request
+    Attributes:
+        UNAGGREGATED_SECTION_HEADER: Str header for section on pull requests where
+        unaggregated chile pull requests are linked to.
+
+    Methods:
+        aggregate_child_change_notes: Aggregates change notes from child prs into parent
+        update_unaggregated_pr_header: Adds a new child pr to the section
+    """
+
+    UNAGGREGATED_SECTION_HEADER = "\r\n\r\n# Unaggregated Pull Requests"
+
+    def __init__(self, github, repo, project_config):
+
+        self.repo = repo
+        self.github = github
+        self.link_pr = True  # create links to pr on parsed change notes
+        self.has_issues = True  # need for parsers
+        self.do_publish = True  # need for parsers
+        self.parser_config = (
+            project_config.project__git__release_notes__parsers.values()
+        )
+        super(ParentPullRequestNotesGenerator, self).__init__()
+
+    def _init_parsers(self):
+        """Invoked from Super Class"""
+        for cfg in self.parser_config:
+            parser_class = import_global(cfg["class_path"])
+            self.parsers.append(parser_class(self, cfg["title"]))
+
+        # New parser to collect developer notes above tracked headers
+        self.parsers.append(GithubLinesParser(self, "Notes From Child PRs"))
+        self.parsers[-1]._in_section = True
```

Comment:
With this here, is https://github.com/SFDO-Tooling/CumulusCI/pull/1189/files#diff-df899217f555b1934861052e5487ca3dR61 still necessary?